src/app/record/components/top-bar/top-bar.component.scss

```diff
@@ -103,6 +103,14 @@ app-top-bar-actions {
       .other-names-wrapper {
         padding: 16px;
 
+        p {
+          margin: 0px;
+        }
+
+        .comma {
+          margin-left: -0.3em;
```

Comment:
I think this won't be working very well on RTL languages. 
Should we maybe use Angular's `preserverWhiteSpaces` on the component decorator?

Reply:
I applied another solution